Lib/CollectionViewPagingLayout.swift

```diff
@@ -43,7 +43,13 @@ public class CollectionViewPagingLayout: UICollectionViewLayout {
     }
     
     private var numberOfItems: Int {
-        collectionView?.numberOfItems(inSection: 0) ?? 0
+        var count = 0;
```

Comment:
```suggestion
        var count = 0
```
you can install [SwiftLint](https://github.com/realm/SwiftLint) and then it will format the code for you